src/components/bcros/businessDetails/Links.vue

```diff
@@ -0,0 +1,174 @@
+              alt=""
+              class="pa-1"
+            >
+          </template>
+          <span class="font-13 ml-1">{{ $t('button.tombstone.businessSummary') }}</span>
+        </UButton>
+      </BcrosTooltip>
+    </span>
+
+    <div class="mb-2">
+      <BcrosBusinessDetailsLinkActions v-if="!isDisableNonBenCorps && currentBusiness.identifier" />
+    </div>
+  </div>
+</template>
+
+<script setup lang="ts">
+import { CorpTypeCd, FilingTypes } from '@bcrs-shared-components/enums'
+import type { DocumentI } from '~/interfaces/document-i'
+import { fetchDocuments } from '~/services/legal-s'
+import { BusinessStateE } from '~/enums/business-state-e'
+
+const { currentBusiness } = storeToRefs(useBcrosBusiness())
+const { getStoredFlag } = useBcrosLaunchdarkly()
+const { isAllowedToFile } = useBcrosBusiness()
+const { isFirm } = storeToRefs(useBcrosBusiness())
+
+const enableNonBenCorpsFlag = computed(() => getStoredFlag('enable-non-ben-corps'))
+const isAllowedBusinessSummary = computed(() =>
+  currentBusiness.value.identifier &&
+  !!getStoredFlag('supported-business-summary-entities')?.includes(currentBusiness.value.legalType)
+)
+
+const isDisableNonBenCorps = computed(() => {
+  if ([CorpTypeCd.BENEFIT_COMPANY, CorpTypeCd.BC_ULC_COMPANY, CorpTypeCd.BC_CCC]
+    .includes(currentBusiness.value.legalType)) {
+    return !enableNonBenCorpsFlag
+  }
+  return false
+})
+
+const isPendingDissolution = computed(() => {
+  return false
+  // todo: implement !!FUTURE not implemented in current dashboard
+})
+
+const isChangeBusinessInfoDisabled = computed(() => {
+  // todo: add staff exclusion. Staff should be allowed when business is not in good standing
+  const isAllowed =
+    // if it's coop
+    (currentBusiness.value.legalType === CorpTypeCd.COOP &&
+      !!getStoredFlag('special-resolution-ui-enabled') &&
+      isAllowedToFile(FilingTypes.SPECIAL_RESOLUTION)) ||
+    // if it's firm
+    (isFirm && isAllowedToFile(FilingTypes.CHANGE_OF_REGISTRATION)) ||
+    // otherwise
+    isAllowedToFile(FilingTypes.ALTERATION)
+
+  return !currentBusiness.value.goodStanding || isAllowed
+})
+const promptChangeBusinessInfo = () => {
+  // todo: implement
+
+  navigateTo('https://www.corporateonline.gov.bc.ca/', { external: true })
+}
+
+/** Request and Download Business Summary Document. */
+const downloadBusinessSummary = async (): Promise<void> => {
+  // todo: add loading full screen // ticket #22059
+  // this.setFetchingDataSpinner(true)
+  const businessId = currentBusiness.value.identifier
+  const apiURL = useRuntimeConfig().public.legalApiURL
+  const summaryDocument: DocumentI = {
+    title: 'Summary',
+    filename: `${businessId} Summary - ${todayIsoDateString()}.pdf`,
+    link: `${apiURL}/businesses/${businessId}/documents/summary`
+  }
+
+  const blob = await fetchDocuments(summaryDocument.link) // todo: show alert box on error
+  if (blob) {
+    if (window.navigator && window.navigator.msSaveOrOpenBlob) {
```

Comment:
can you use the 'download-file' util ? I think it is the same code. NOTE: downloadFile is used by director search as well

Reply:
sorry I remember you asked if we had something like this before haha

Reply:
OK, moved the code a bit, and used the util file to download and save it.